language/python/develop.md

```diff
@@ -4,8 +4,253 @@ keywords: python, local, development, run,
 description: Learn how to develop your application locally.
 ---
 
-> ### Coming soon
->
-> Thanks for your interest in learning how to containerize a Python application. We are actively working on this topic. Watch this space!
+{% include_relative nav.html selected="3" %}
 
-<br />
+## Prerequisites
+
+Work through the steps to build an image and run it as a containerized application in [Run your image as a container](run-containers.md).
+
+## Introduction
+
+In this module, we’ll walk through setting up a local development environment for the application we built in the previous modules. We’ll use Docker to build our images and Docker Compose to make everything a whole lot easier.
+
+In this module, we’ll walk through setting up a local development environment for the application we built in the previous modules. We’ll use Docker to build our images and Docker Compose to make everything a whole lot easier.
+
+## Local database and containers
```

Comment:
Should we change section title to `Running a database in a container/ containerized database` or something else to quickly describe what we talk about in this part? And maybe we can move the first paragraph in the intro part.